variables.tf

```diff
@@ -1,9 +1,90 @@
-########################################################################################################################
+##############################################################################
 # Input Variables
-########################################################################################################################
+##############################################################################
 
-#variable "my_variable" {
-#  type        = string
-#  description = "A description of my variable"
-#  default     = "default_value"
-#}
+variable "ibmcloud_api_key" {
+  description = "APIkey that's associated with the account to use, set via environment variable TF_VAR_ibmcloud_api_key"
```

Comment:
We should no mention `set via environment variable TF_VAR_ibmcloud_api_key` as this is not applicable in Projects / Schematics

Reply:
removed